### ISSUE TITLE
ModuleNotFoundError: No module named 'zoneinfo'

### ISSUE THREAD

**Issue by theaga-d** [python/cpython#109973]
# Bug report

### Bug description:

getting following error while importing O365

```pytb
    from O365 import *
  File "/home/azureuser/actions-runner/_work/_tool/Python/3.7.17/x64/lib/python3.7/site-packages/O365/__init__.py", line 9, in <module>
    from .account import Account
  File "/home/azureuser/actions-runner/_work/_tool/Python/3.7.17/x64/lib/python3.7/site-packages/O365/account.py", line 1, in <module>
    from .connection import Connection, Protocol, MSGraphProtocol, MSOffice365Protocol
  File "/home/azureuser/actions-runner/_work/_tool/Python/3.7.17/x64/lib/python3.7/site-packages/O365/connection.py", line 17, in <module>
    from zoneinfo import ZoneInfoNotFoundError
ModuleNotFoundError: No module named 'zoneinfo'
```

### CPython versions tested on:

3.8

### Operating systems tested on:

Linux

**Comment by AA-Turner** [comment]
Your traceback indicates that you're actually using Python 3.7, which is end-of-life. Regardless, ``zoneinfo`` was only implemented in Python 3.9, which explains your errors. Please update to a newer version of Python.

https://docs.python.org/3/library/zoneinfo.html

A

**Comment by hugovk** [comment]
There's also a backport for Python 3.6 and 3.7: https://pypi.org/project/backports.zoneinfo/

But I also recommend upgrading to a newer version of Python, 3.7 no longer receives security updates: https://devguide.python.org/versions/
